examples/README.md

```diff
@@ -29,3 +29,5 @@ go get github.com/kubernetes-sigs/kustomize
 
  * [breakfast](breakfast.md) - Customize breakfast for
    Alice and Bob.
+   
+ * [wordpress](wordpress/README.md) - Create wordpress with mysql
```

Comment:
Should emphasize var substitution here more (that's the feature), and de-emphasize wordpress (that's an implementation detail).

perhaps:

 * [container args](containerArgs/README.) - Injecting k8s runtime data into container arguments (e.g. to point  _wordpress_ to a _SQL_ service).